lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/service/LessonBuilderEntityProducer.java

```diff
@@ -1044,14 +1045,19 @@ public String merge(String siteId, Element root, String archivePath, String from
 		     Long oldPageId = Long.valueOf(oldPageIdString);
 		     SimplePage page = simplePageToolDao.makePage("0", siteId, title, 0L, 0L);
 		     String gradebookPoints = pageElement.getAttribute("gradebookpoints");
-		     if (gradebookPoints != null && !gradebookPoints.equals("")) {
+		     if (StringUtils.isNotEmpty(gradebookPoints)) {
 			 page.setGradebookPoints(Double.valueOf(gradebookPoints));
 		     }
 		     String folder = pageElement.getAttribute("folder");
-		     if (folder != null && !folder.equals(""))
+		     if (StringUtils.isNotEmpty(folder))
 			 page.setFolder(folder);
+		     // Carry over the custom CSS sheet if present. These are of the form
+		     // "/group/SITEID/LB-CSS/whatever.css", so we need to map the SITEID
+		     String cssSheet = pageElement.getAttribute("csssheet");
+		     if (StringUtils.isNotEmpty(cssSheet))
+			 page.setCssSheet(cssSheet.replace("/group/"+fromSiteId+"/", "/group/"+siteId+"/"));
```

Comment:
This should "probably" just be a replaceFirst, anything else if they actually put it in there wouldn't actually be changed.